src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerTypeSymbol.cs

```diff
@@ -83,24 +84,22 @@ internal override bool Equals(TypeSymbol t2, TypeCompareKind compareKind, IReadO
                 return false;
             }
 
-            return _signature.Equals(other._signature, compareKind, isValueTypeOverrideOpt);
+            return Signature.Equals(other.Signature, compareKind, isValueTypeOverrideOpt);
         }
 
         public override int GetHashCode()
         {
-            return Hash.Combine(1, _signature.GetHashCode());
+            return Hash.Combine(1, Signature.GetHashCode());
         }
 
         protected override ISymbol CreateISymbol()
         {
-            // PROTOTYPE(func-ptr): Implement
-            throw new NotImplementedException();
+            return CreateITypeSymbol(CodeAnalysis.NullableAnnotation.None);
```

Comment:
>return CreateITypeSymbol [](start = 12, length = 24)

I think the code reuse doesn't give us a lot of benefits, but disallows us to assert that we properly use internal cash for public symbols.